.github/ISSUE_TEMPLATE/new_component.md

```diff
@@ -0,0 +1,63 @@
+---
+name: New Component
+about: Interested in contributing a new component to Fluent UI React-Native? This template includes necessary information to get started, and steps to completion
+---
+
+<!-- Use this template for new components or new component variants -->
+
+## Component details
+
+### Description
+
+<!-- fill this out -->
+
+### Which product teams/scenarios need this control
+
+<!-- The more teams/scenarios that would use this control the better chance it will get prioritized -->
+
+### Design assets
+
+<!-- Please provide links to redlines or screenshots of intended component design -->
+<!-- Are you internal to Microsoft and your design isn't yet public? Please reach out to us internally via the Fluent UI Apple Teams channel or log a request at https://aka.ms/fluentui-native. -->
+
+### Component ownership
+
+<!-- Are there one or more people who can help maintain this component over time? Who will address bugs? -->
+
+## Component composition
+
+### Imports/Dependencies
+
+<!-- What other dependencies will your component be using -->
+
+### Proposed Public API
+
+<!-- Define the proposed public API for this control -->
+
+### Intended Platforms
+- [ ] iOS
+- [ ] macOS
+- [ ] win32
+- [ ] windows
+
+## Component code
+
+### Example code
```

Comment:
It's unclear to me what the difference between these two is - maybe comments would help?

Reply:
Not sure if you're referring to component code vs example code but if so, example code is a subsection of component code so there isn't necessarily a difference

Reply:
It's a lot more clear outside of markdown where the headers are noticeably different sizes

Reply:
Though I could read markdown by now :). The part that's weird to me is that "New Component Checklist" doesn't seem like it fits under "component code". Maybe just "Example code" and "new Component Checklist" are siblings without the need for a common parent?